.github/workflows/main.yml

```diff
@@ -161,10 +161,12 @@ jobs:
           key: ${{ env.EMSDK_CACHE_NUMBER }}-${{ env.EMSCRIPTEN_VERSION }}-${{ runner.os }}
```

Comment:
Let's remove this emsdk caching job too. We can add back caching when we find a good way to cache pyodide-build.